package.json

```diff
@@ -14,14 +14,14 @@
     "storybook": "./web/scripts/run-storybook.sh",
     "storybook-smoke-test": "storybook dev -p 9002 -c web/.storybook --ci --smoke-test",
     "test-storybook": "NODE_TLS_REJECT_UNAUTHORIZED=0 test-storybook -c web/.storybook --url https://localhost:9002 --skipTags=skip-test --browsers=chromium",
-    "test": "jest",
-    "test-coverage": "jest --coverage && web/scripts/print-coverage-link.sh",
+    "test": "make dump-preset-roles && jest",
+    "test-coverage": "make dump-preset-roles && jest --coverage && web/scripts/print-coverage-link.sh",
     "test-update-snapshot": "pnpm run test --updateSnapshot",
-    "tdd": "jest --watch",
+    "tdd": "make dump-preset-roles && jest --watch",
     "lint": "pnpm eslint && pnpm prettier-check",
     "lint-fix": "pnpm eslint --fix && pnpm prettier-write",
     "eslint": "eslint --quiet '+(e|web)/**/*.{ts,tsx,js,jsx,mts}'",
-    "type-check": "NODE_OPTIONS='--max-old-space-size=4096' tsc --build",
+    "type-check": "NODE_OPTIONS='--max-old-space-size=4096' make dump-preset-roles && tsc --build",
```

Comment:
We could avoid needing this file for type checking if we read it from disk in the test file directly instead of importing it.